kong/init.lua

```diff
@@ -90,8 +91,56 @@ local set_current_peer = ngx_balancer.set_current_peer
 local set_timeouts     = ngx_balancer.set_timeouts
 local set_more_tries   = ngx_balancer.set_more_tries
 
+local plugins_map_version
+local plugins_map_semaphore
+
+local PLUGINS_MAP_CACHE_OPTS = { ttl = 0 }
```

Comment:
What do you mean by different module?

https://github.com/Kong/kong/blob/perf/plugins-iterator/kong/init.lua#L354-L387

E.g.:
```lua
local ok, err = cache:get("router:version", { ttl = 0 }, function()
    return "init"
  end
```

Could be written, as this repeates in 3 places (ok, api_router is going away) now:
```lua
local ok, err = cache:get("router:version", CACHE_OPTS, CACHE_INIT)
```

And that:
```lua
  plugins_map_version, err = cache:get("plugins_map:version",
                                       PLUGINS_MAP_CACHE_OPTS,
                                       function() return "init" end)
```

changed to
```lua
  plugins_map_version, err = cache:get("plugins_map:version", CACHE_OPTS, CACHE_INIT)
```

But not a big deal. Just repeating here so that we were in a same page.